src/mailadm/mailcow.py

```diff
@@ -1,5 +1,6 @@
 import requests as r
 
+HTTP_TIMEOUT=5
```

Comment:
```suggestion
HTTP_TIMEOUT = 5

```

This should fix the CI complaints btw :)